### DISCUSSION TITLE
Add negation to search

### DISCUSSION THREAD

**Discussion by Coderx7** [civitai/civitai#295] [Ideas]
Hi, It would be great to have negation in the search box i.e. fetch all categories that doest not contain a specific keyword or sets of keywords.

**Comment by dkhold** [comment]
+1000, this would be so useful to filter out entire categories of content that I'm not interested in.

**Comment by AffableEffable** [comment]
Is this still not a thing? It seems obvious to me that if I want to search for, say, "War" but exclude things from "Warhammer" I should be able to do that pretty easily in the search box without any filters or whatever set up previously. I am new to this site and SD in general, so am I just missing how to do this?
